modules/contact/src/problems/AugmentedLagrangianContactProblem.C

```diff
@@ -40,15 +40,14 @@ AugmentedLagrangianContactProblem::validParams()
 
 AugmentedLagrangianContactProblem::AugmentedLagrangianContactProblem(const InputParameters & params)
   : ReferenceResidualProblem(params),
-    _num_lagmul_iterations(0),
-    _max_lagmul_iters(getParam<int>("maximum_lagrangian_update_iterations"))
+    _maximum_lagrangian_update_iterations(getParam<int>("maximum_lagrangian_update_iterations"))
```

Comment:
Not your doing, but curious why this is a signed integer

Reply:
I'll make it unsigned